Nothing I did could've caused this issue. Something up with jenkins?

```
03:13:30 
03:13:30 Exception occurred:
03:13:30   File "/var/jenkins/workspace/PR/salt-pr-docs-n/salt/utils/yamldumper.py", line 53, in <module>
03:13:30     class IndentedSafeOrderedDumper(IndentMixin, SafeOrderedDumper):
03:13:30 TypeError: Error when calling the metaclass bases
03:13:30     metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
03:13:30 The full traceback has been saved in /tmp/sphinx-err-QvmR66.log, if you want to report the issue to the developers.
03:13:30 Please also report this if it was a user error, so that a better error message can be provided next time.
03:13:30 A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
03:13:30 Failed to import 'salt.modules.cytest': no module named salt.modules.cytest
03:13:30 make: *** [html] Error 1
03:13:30 Build step 'Execute shell' marked build as failure
03:13:30 Performing Post build task...
03:13:30 Match found for :. : True
03:13:30 Logical operation result is TRUE
03:13:30 Running script  : github-commit-status --auth-token=${GITHUB_AUTH_TOKEN} --context=jenkins/${JOB_NAME} --target-url=${BUILD_URL} ${SALT_PR_GIT_COMMIT} || echo "Failed to set the commit state"
03:13:30 [salt-pr-docs-n] $ /bin/sh -xe /tmp/hudson3434449460586606560.sh
03:13:30 + github-commit-status --auth-token=**** --context=jenkins/PR/salt-pr-docs-n --target-url=https://jenkins.saltstack.com/job/PR/job/salt-pr-docs-n/8721/ 713627863f1c9d41a6458f511fedf3da09b220e9
03:13:31 POST BUILD TASK : SUCCESS
03:13:31 END OF POST BUILD TASK : 0
03:13:31 [BFA] Scanning build for known causes...
03:13:31 [BFA] No failure causes found
03:13:31 [BFA] Done. 0s
03:13:31 Notifying upstream projects of job completion
03:13:31 Finished: FAILURE
```

--- a/salt/modules/zoneadm.py
+++ b/salt/modules/zoneadm.py
@@ -62,7 +62,7 @@ def __virtual__():
     Solaris 10, OmniOS, OpenIndiana, OpenSolaris, or Smartos.
     '''
     if _is_globalzone() and salt.utils.which('zoneadm'):
-        if __grains__['os_family'] == 'Solaris':
+        if __grains__['os'] in ['Oracle Solaris', 'OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
             return __virtualname__
 
     return (

--- a/salt/modules/zoneadm.py
+++ b/salt/modules/zoneadm.py
@@ -62,9 +62,10 @@ def __virtual__():
     Solaris 10, OmniOS, OpenIndiana, OpenSolaris, or Smartos.
     '''
     if _is_globalzone() and salt.utils.which('zoneadm'):
-        if __grains__['os'] in ['Oracle Solaris', 'OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
+        if __grains__['os'] in ['OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
             return __virtualname__
-
+        elif __grains__['os'] == 'Oracle Solaris' and int(__grains__['osmajorrelease']) == 10:
+            return __virtualname__
     return (
         False,
         '{0} module can only be loaded in a solaris globalzone.'.format(

--- a/salt/modules/zonecfg.py
+++ b/salt/modules/zonecfg.py
@@ -98,7 +98,7 @@ def __virtual__():
     Solaris 10, OmniOS, OpenIndiana, OpenSolaris, or Smartos.
     '''
     if _is_globalzone() and salt.utils.which('zonecfg'):
-        if __grains__['os_family'] == 'Solaris':
+        if __grains__['os'] in ['Oracle Solaris', 'OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
             return __virtualname__
 
     return (

--- a/salt/modules/zonecfg.py
+++ b/salt/modules/zonecfg.py
@@ -98,9 +98,10 @@ def __virtual__():
     Solaris 10, OmniOS, OpenIndiana, OpenSolaris, or Smartos.
     '''
     if _is_globalzone() and salt.utils.which('zonecfg'):
-        if __grains__['os'] in ['Oracle Solaris', 'OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
+        if __grains__['os'] in ['OpenSolaris', 'SmartOS', 'OmniOS', 'OpenIndiana']:
             return __virtualname__
-
+        elif __grains__['os'] == 'Oracle Solaris' and int(__grains__['osmajorrelease']) == 10:
+            return __virtualname__
     return (
         False,
         '{0} module can only be loaded in a solaris globalzone.'.format(

